cmd/utils/customflags.go

```diff
@@ -46,7 +44,6 @@ func (self *DirectoryString) Set(value string) error {
 // Custom cli.Flag type which expand the received string to an absolute path.
 // e.g. ~/.ethereum -> /home/username/.ethereum
 type DirectoryFlag struct {
-	cli.GenericFlag
```

Comment:
If it was pending on Dec 11, and today it's Jan 11, then if Dec 11 was the first day of pending, today is the 32nd. That's more than a month by any measure :D